WireDomain/Sources/WireDomain/Repositories/UpdateEvents/UpdateEventsLocalStore.swift

```diff
@@ -113,36 +113,43 @@ final class UpdateEventsLocalStore: UpdateEventsLocalStoreProtocol {
 
     public func fetchStoredEventEnvelopes(
         limit: UInt
-    ) async throws -> [UpdateEventEnvelope] {
-        try await eventContext.perform { [eventContext, updateEventCoder] in
+    ) async throws -> [StoredUpdateEventEnvelope] {
```

Comment:
suggestion: return a [(NSManagedObjectID, UpdateEventEnvelope)] so we keep the updateEventCoder inside the localstore

what do you think?